moon/controller_round1.py

```diff
@@ -13,17 +13,24 @@ def on_object_reached(self, data):
         object_type = data
         x,y,z = self.xyz
         print(self.sim_time, "app: Object %s reached" % object_type)
-        response = self.send_request('artf %s %f %f 0.0\n' % (object_type, x, y)).decode("ascii")
-        print(self.sim_time, "app: Volatile report response: %s" % response)
-        if response == 'ok':
-            pass
-        else:
-            # do nothing, ie keep going around and try to match the view
-            pass
+
+
+        def process_volatile_response(response):
+            print(self.sim_time, "app: Volatile report response: %s" % response)
+            if response == 'ok':
```

Comment:
if this string `ok` is the problem vs. `b'ok'` then I would introduce some constant RESPONSE_OK (?) and define it in controller as binary ok??